Makefile

```
@@ -241,13 +235,22 @@ rtk-rewrite-sync: ## Sync rtk-rewrite.sh from upstream rtk repo.
 	@echo "✅ rtk-rewrite.sh synced"
 
-.PHONY: upgrade-dev
-upgrade-dev: ## Upgrade inside the Nix dev shell (mirrors CI).
-	@echo "🔄 Running upgrade inside the Nix dev shell..."
-	@DEVENV_ROOT=$(CURDIR) $(NIX_ALLOW_UNFREE) $(NIX_EXEC) develop $(NIX_FLAGS) .# --command $(MAKE) upgrade
+##@ Update
 
-.PHONY: dev
-dev: nix-develop ## Enter the Nix dev shell (alias for nix-develop).
+.PHONY: uptate
+update: nix-update overlays-update neovim-update llm-update gitalias-update ## Update Nix flake, overlays, Neovim plugins, LLM configs, and gitalias
```

checkmake (0.2.2)

[warning] Target "update" should be declared PHONY.

(phonydeclared)